composer.json

```
@@ -16,7 +16,7 @@
         "ext-dom": "*",
         "ext-libxml": "*",
         "inpsyde/assets": "^2.8",
-        "inpsyde/fau-degree-program-common": "*",
+        "rrze-webteam/fau-studium-common": "*",
```

As per https://github.com/RRZE-Webteam/FAU-Studium-Common/pull/1#discussion_r1582594223.
```suggestion
        "rrze/fau-studium-common": "*",
```